http_service/Dockerfile

```diff
@@ -11,8 +11,6 @@ COPY . /code/bugbug_http_service/
 # Load the models
 WORKDIR /code/
 
-COPY ./models /code/models
-
 ARG CHECK_MODELS
```

Comment:
We don't need CHECK_MODELS and ensure_models.sh anymore here, right? Given that we move the classification in the bg part.